packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -1159,6 +1159,60 @@ describe('Checkout', () => {
                 });
             });
         });
+
+        describe('shouldShowAgeVerificationForm ::', () => {
+            describe('when the `DATE_OF_BIRTH_REQUIRED` issue exists in errors', () => {
+                let wrapper;
+                beforeEach(() => {
+                    // Arrange && Act
+                    wrapper = shallowMount(VueCheckout, {
+                        store: createStore({
+                            ...defaultCheckoutState,
+                            errors: [
+                                {
+                                    code: 'DATE_OF_BIRTH_REQUIRED',
+                                    shouldShowInDialog: false
+                                }
+                            ]
+                        }),
+                        i18n,
+                        localVue,
+                        propsData,
+                        mocks: {
+                            $cookies
+                        }
+                    });
+                });
+
+                it('should return true', () => {
+                    // Assert
+                    expect(wrapper.vm.shouldShowAgeVerificationForm).toBe(true);
+                });
+            });
+
+            describe('when the `DATE_OF_BIRTH_REQUIRED` issue does not exist in errors', () => {
+                let wrapper;
+                beforeEach(() => {
+                    // Arrange && Act
+                    wrapper = shallowMount(VueCheckout, {
+                        store: createStore({
+                            ...defaultCheckoutState
```

Comment:
I don't think you need this or the cookie mock?

Reply:
I've removed the cookie mock but doesn't all mounting of the checkout require the mocked store?

Reply:
It does but you don't need the `{...defaultState}` part just `store: createStore()`